src/ontology/pato-edit.obo

```diff
@@ -14804,7 +14804,7 @@ subset: mpath_slim
 subset: value_slim
 is_a: PATO:0002067 ! stratification
 relationship: is_opposite_of PATO:0002068 ! stratified
-created_by: George Gkoutos
+property_value: http://purl.org/dc/elements/1.1/contributor https://orcid.org/0000-0002-2061-091X
```

Comment:
Let's use `http://purl.org/dc/terms/contributor`. What you have is not wrong, but this is the recommended one.

Check: https://dublincore.org/specifications/dublin-core/dcmi-terms/#contributor

Reply:
Great!